app/helpers/orchid/date_helper.rb

```diff
@@ -0,0 +1,75 @@
+module Orchid::DateHelper
+
+  def clear_dates_params
+    # duplicate parameters, remove date fields, and return
+    options = params.permit!.deep_dup
+    options.delete("date_from")
+    options.delete("date_to")
+    return options
+  end
+
+  # TODO could abstract this into a CDRH gem
+  def date_default(date_params, default_date)
+    y, m, d = date_params.map { |x| x.gsub(/\s+/, "") }
+    y = default_date[0] if y.blank?
+    m = default_date[1] if m.blank?
+    d = default_date[2] if d.blank?
+
+    # solr can't handle months and days without zero padding (01)
+    m = "0#{m}" if m.length == 1
+    d = "0#{d}" if d.length == 1
+
+    return "#{y}-#{m}-#{d}"
+  end
+
+  # main method responsible for taking params and returning
+  # a set of options for querying the API
+  def date_filter aParams=params
+    options = params.permit!.deep_dup
+    # date search
+    if (!options["date_from"].nil? && !options["date_from"].join('').empty?) \
```

Comment:
Looks good to me.  Do you want me to change it in L&C, too?

https://github.com/CDRH/lewisandclark/commit/162048b06d3d216af6ae94dbb8472db24635c3e4